fix.sh

```diff
@@ -136,18 +136,22 @@ ensure_bundle() {
   # update --conservative' flag works as expected - important when
```

Comment:
[bigfiles] This file, one of the 3 largest in the project, increased in size to 370 lines.  The total size of those files is now 746 lines (target: 696).  Is this file complex enough to refactor?